server/src/main/java/com/google/musicanalysis/site/AnalysisServlet.java

```diff
@@ -21,23 +21,32 @@
   protected void doGet(HttpServletRequest req, HttpServletResponse res)
       throws ServletException, IOException {
 
-    String videoName = req.getParameter("name");
+    String input = req.getParameter("name");
 
-    assert videoName == null : "Something went wrong with sending to backend.";
+    assert input == null : "Something went wrong with sending to backend.";
```

Comment:
throw errors instead? (Though I guess this is a type of error...)

Reply:
`assert` statements are almost the same as throwing an error. The main difference is that `assert` statements can be ignored when running in production, which means that `input` will not be checked for `null` when you deploy. All of that can be configured, but watch out.

Reply:
I'll just check on the front-end for more clarity.